CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+##Contributing to this repository
+
+Welcome!  Thanks for your interest in contributing to Software Carpentry's introductory shell lesson.  
+
+###Intro
+
+Software Carpentry is an open source project, and we welcome contributions of all kinds: new and improved lessons, bug reports, and small fixes to existing material are all useful.
+
+By contributing, you are agreeing that Software Carpentry may redistribute your work under these licenses.
+
+###Making and Submitting Changes
+
+Don't know where to start?  Please see the current list of issues (linked in the right sidebar) for ideas for contributing to this repository. 
+
+For making your contribution, we use the GitHub flow, which is nicely explained in the chapter [Contributing to a Project in Pro Git by Scott Chacon](http://git-scm.com/book/en/v2/GitHub-Contributing-to-a-Project).  For this repository, you should branch from and then submit a pull request against the `gh-pages` branch instead of `master`.  
+
+When editing lesson pages, you need only change and commit the `.md` source file.  
+
+Even if you don't have any changes to make, we welcome comments on  pull requests, or discussion among issues.  
+
+###Other Resources
+
+This lesson is based on the template found at  https://github.com/swcarpentry/lesson-template.  Visit that repository for instructions on formatting, building, and submitting Software Carpentry lessons, or run make in this directory for a list of helpful commands.
+
+###Contact
+
+The maintainers for this repository are: 
+
+@gdevenyi
```

Comment:
Maybe

~~~
- [Gabriel A. Devenyi][gdevenyi]
- [Christina Koch][ChristinaLK]

[gdevenyi]: https://github.com/gdevenyi
[ChristinaLK]: https://github.com/ChristinaLK
~~~

This will make easy to read this file offline.